GRPCServiceA/pkg/user/endpoints.go

```
@@ -47,3 +51,31 @@ func makeCreateUserEndpoint(s Service) endpoint.Endpoint {
 		return createResp, nil
 	}
 }
+
+func makeDeleteUserEndpoint(s Service) endpoint.Endpoint {
+	return func(ctx context.Context, request interface{}) (interface{}, error) {
+		req, ok := request.(ent.DeleteUserReq)
+		if !ok {
+			return nil, errors.New("error asserting DeleteUserReq")
```

You could return an error from the error package...